tutorial/arrow-integration-malibu-advanced.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Arrow Integration to Malibu Advanced
+parent: Malibu Advanced Tutorial
+nav_order: 1
+---
+# {{page.title}}
+
+_This tutorial was contributed by [Amogh](https://github.com/ags1773) and [Nandakishore Prakash Rao](https://twitter.com/nkp_adm)_
+
+Apart from building custom components on the Malibu Advanced app, a wide range of components that are built as a part of the [Arrow](https://developers.quintype.com/quintype-node-arrow/) can also be included and customized based on the requirements.
+
+The list of the Arrow components can be found [here](https://developers.quintype.com/quintype-node-arrow/).
+
+As you clone the Malibu Advanced repository, you will be getting some of the Arrow components out of the box.
+
+Ex: ElevenStories, FourColTwelveStories, etc.
+
+**Steps to add a new Arrow component to your app:**
```

Comment:
Currently, this feature is available in Malibu-Advanced. Could you please mention that if they cloned before  DD/MM  they need to follow the below steps to integrate the arrow?